src/test/java/io/vacco/oruzka/OzHashSpec.java

```diff
@@ -10,14 +10,19 @@
 import java.util.stream.IntStream;
 
 import static j8spec.J8Spec.*;
+import static org.junit.Assert.*;
 
 @RunWith(J8SpecRunner.class)
 public class OzHashSpec {
   static {
     describe("Hashing", () -> {
       it("can hash an array of strings", () -> {
-        OzArrays.toStringConcat("Hello", "World")
-            .ifPresent(bytes -> System.out.println(OzMd5.md5SumOf(bytes)));
+        Object[] strings = new Object[] {"Hello", "World"};
+        OzArrays.toStringConcat(strings)
+            .ifPresent(bytes -> {
+              String md5 = OzMd5.md5SumOf(bytes);
```

Comment:
Can we change this into a `var` style declaration? Thanks

Reply:
Ok but why do we need to do that?

Reply:
To improve code readability. Thanks

Reply:
I think it looks good now.